### PR TITLE
Tasks16-30

### PR DESCRIPTION
**Pull request by nazar-ginailo**
New resolved tasks from 16 to 30.

### CODE REVIEW

File: tasks/20_truncate-array/truncate-array.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export function truncateArray(arr, n) {
+  arr.splice(0, n);
+  return arr;
```
Comment (Tsvetkovv) [inline tasks/20_truncate-array/truncate-array.js]: please do not mutate original array

File: tasks/21_check-letters/check-letters.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export function checkLetters(arr) {
+  const firstStr = arr[0].toLowerCase();
+  const secondStr = arr[1].toLowerCase();
+  for (const item of secondStr) {
+    if (firstStr.indexOf(item) === -1) {
+      return false;
+    }
+  }
+  return true;
+}
```
Comment (Tsvetkovv) [inline tasks/21_check-letters/check-letters.js]: maybe split to array and use `every` method?

Comment (Tsvetkovv) [reply tasks/21_check-letters/check-letters.js]: and `include`

File: tasks/25_rot13-encoder/rot13-encoder.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function rot13Encoder(str) {
+  const input = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+  const output = 'NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm';
+  let index = (x) => input.indexOf(x);
+  let translate = (x) => index(x) > -1 ? output[index(x)] : x;
+  return str.split('').map(translate).join('');
+}
```
Comment (Tsvetkovv) [inline tasks/25_rot13-encoder/rot13-encoder.js]: please try to operate with offsets, see useful methods in description

File: tasks/28_vowel-count/vowel-count.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export function getVowelCount(str) {
+  return str.split('').filter((letter) => 'aeiouAEIOU'.includes(letter)).length;
```
Comment (Tsvetkovv) [inline tasks/28_vowel-count/vowel-count.js]: good practice would extract value `aeiou` to variable with meaningful name

File: tasks/29_remove-duplicates/remove-duplicates.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export function removeDuplicates(arr) {
+  return arr.filter(function(item, pos) {
+    return arr.indexOf(item) == pos;
+  });
+}
+
+// return [...new Set(arr)]; - it's working,
+// but i have problem with eslint,
+// Maybe problem with ES6 syntax.
+//  env: {es6: true}
```
Comment (ArtemLunt) [inline tasks/29_remove-duplicates/remove-duplicates.js]: This way seems better from complexity perspective.
Pls fix eslint configuration and use this approach.

File: tasks/27_basic-subclasses/basic-subclasses.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    this.name = name;
+  }
+}
+
+export class Woman extends Human {
+  constructor(name) {
+    super('female');
+    this.name = name;
+  }
+}
```
Comment (ArtemLunt) [inline tasks/27_basic-subclasses/basic-subclasses.js]: Let's move name to Human class.

**Pull request merged**